src/ColorPicker.test.js

```diff
@@ -204,31 +204,66 @@ describe('ColorPicker', () => {
-			const colorHslLightnessInput = wrapper.find(`#${id}-color-hsl-l`)
+			const colorHslLightnessLabel = wrapper.find(`#${id}-color-hsl-l`)
+			expect(colorHslLightnessLabel.exists()).toBe(true)
+			const colorHslLightnessInput = wrapper.find(`#${id}-color-value-hsl-l`)
 			expect(colorHslLightnessInput.exists()).toBe(true)
-			const colorHslAlphaInput = wrapper.find(`#${id}-color-hsl-a`)
+			const colorHslAlphaLabel = wrapper.find(`#${id}-color-hsl-a`)
+			expect(colorHslAlphaLabel.exists()).toBe(true)
+			const colorHslAlphaInput = wrapper.find(`#${id}-color-value-hsl-a`)
 			expect(colorHslAlphaInput.exists()).toBe(true)
 
-			const formatSwitchButton = wrapper.find('.vacp-format-switch-button')
 			await formatSwitchButton.trigger('click')
 
-			const colorRgbRedInput = wrapper.find(`#${id}-color-hwb-h`)
+			const colorHwbRedLabel = wrapper.find(`#${id}-color-hwb-h`)
+			expect(colorHwbRedLabel.exists()).toBe(true)
+			const colorHwbRedInput = wrapper.find(`#${id}-color-value-hwb-h`)
+			expect(colorHwbRedInput.exists()).toBe(true)
+			const colorHwbGreenLabel = wrapper.find(`#${id}-color-hwb-w`)
+			expect(colorHwbGreenLabel.exists()).toBe(true)
+			const colorHwbGreenInput = wrapper.find(`#${id}-color-value-hwb-w`)
+			expect(colorHwbGreenInput.exists()).toBe(true)
+			const colorHwbBlueLabel = wrapper.find(`#${id}-color-hwb-b`)
+			expect(colorHwbBlueLabel.exists()).toBe(true)
+			const colorHwbBlueInput = wrapper.find(`#${id}-color-value-hwb-b`)
+			expect(colorHwbBlueInput.exists()).toBe(true)
+			const colorHwbAlphaLabel = wrapper.find(`#${id}-color-hwb-a`)
+			expect(colorHwbAlphaLabel.exists()).toBe(true)
+			const colorHwbAlphaInput = wrapper.find(`#${id}-color-value-hwb-a`)
+			expect(colorHwbAlphaInput.exists()).toBe(true)
+
+			await formatSwitchButton.trigger('click')
+
+			const colorRgbRedLabel = wrapper.find(`#${id}-color-rgb-r`)
+			expect(colorRgbRedLabel.exists()).toBe(true)
+			const colorRgbRedInput = wrapper.find(`#${id}-color-value-rgb-r`)
 			expect(colorRgbRedInput.exists()).toBe(true)
-			const colorRgbGreenInput = wrapper.find(`#${id}-color-hwb-w`)
+			const colorRgbGreenLabel = wrapper.find(`#${id}-color-rgb-g`)
+			expect(colorRgbGreenLabel.exists()).toBe(true)
+			const colorRgbGreenInput = wrapper.find(`#${id}-color-value-rgb-g`)
 			expect(colorRgbGreenInput.exists()).toBe(true)
-			const colorRgbBlueInput = wrapper.find(`#${id}-color-hwb-b`)
+			const colorRgbBlueLabel = wrapper.find(`#${id}-color-rgb-b`)
+			expect(colorRgbBlueLabel.exists()).toBe(true)
+			const colorRgbBlueInput = wrapper.find(`#${id}-color-value-rgb-b`)
 			expect(colorRgbBlueInput.exists()).toBe(true)
-			const colorRgbAlphaInput = wrapper.find(`#${id}-color-hwb-a`)
+			const colorRgbAlphaLabel = wrapper.find(`#${id}-color-rgb-a`)
+			expect(colorRgbAlphaLabel.exists()).toBe(true)
+			const colorRgbAlphaInput = wrapper.find(`#${id}-color-value-rgb-a`)
 			expect(colorRgbAlphaInput.exists()).toBe(true)
```

Comment:
**Suggestion**: Could you please use the following instead of lines 214-267? It's quite repetitive currently. I'd add an actual GitHub suggestion, but GitHub doesn't allow suggesting on lines that weren't changed or that were deleted.

```js
			const formats = ['hsl', 'hwb', 'rgb']

			for (const format of formats) {
				const channels = format.split('')
				expect(wrapper.find(`[id="${id}-color-${format}-${channels[0]}"]`).exists()).toBe(true)
				expect(wrapper.find(`[for="${id}-color-${format}-${channels[0]}"]`).exists()).toBe(true)
				expect(wrapper.find(`[id="${id}-color-${format}-${channels[1]}"]`).exists()).toBe(true)
				expect(wrapper.find(`[for="${id}-color-${format}-${channels[1]}"]`).exists()).toBe(true)
				expect(wrapper.find(`[id="${id}-color-${format}-${channels[2]}"]`).exists()).toBe(true)
				expect(wrapper.find(`[for="${id}-color-${format}-${channels[2]}"]`).exists()).toBe(true)
				expect(wrapper.find(`[id="${id}-color-${format}-a"]`).exists()).toBe(true)
				expect(wrapper.find(`[for="${id}-color-${format}-a"]`).exists()).toBe(true)

				await formatSwitchButton.trigger('click')
			}
```